src/platform/utilities/sso/loginAttempted.js

```diff
@@ -0,0 +1,15 @@
+import localStorage from 'platform/utilities/storage/localStorage';
+
+/*
+ * When set, the `loginAttempted` indicates that the current unauthenticated
+ * user will have to "manually" authenticate against SSOe in order to establish a
+ * session, as a previous attempt may have failed so we need to prevent Auto Login
+ * attempts.
+ */
+export const getLoginAttempted = () => localStorage.getItem('loginAttempted');
+
+export const setLoginAttempted = () =>
+  localStorage.setItem('loginAttempted', true);
+
+export const removeLoginAttempted = () =>
+  localStorage.removeItem('loginAttempted');
```

Comment:
I think this needs to get added to the `logout` function, too. That way it doesn't block attempts after a session is terminated

Reply:
I don't think `loginAttempted` should ever be set when the user is currently logged in